completions/dpkg

```diff
@@ -13,20 +13,20 @@ if _comp_have_command grep-status; then
 else
     _comp_xfunc_dpkg_installed_packages()
     {
-        command grep -A 1 "Package: ${1-}" /var/lib/dpkg/status 2>/dev/null |
-            command grep -B 1 -Ee "ok installed|half-installed|unpacked| \
-            half-configured" \
-                -Ee "^Essential: yes" |
-            awk "/Package: ${1-}/ { print \$2 }" 2>/dev/null
+        command awk -F '\n' -v RS='' "
+            \$1 ~ /Package: $1/ &&
```

Comment:
Would likely be a regression, error in `nounset` mode.
```suggestion
            \$1 ~ /Package: ${1-}/ &&
```

Reply:
Even better, to protect against metachars in the input, we could do away with the regex here:

```suggestion
            index(\$1, \"Package: ${1-}\") == 1 &&
```
(Not a regression in this PR, but I think we could improve this while at it.)